spot_wrapper/spot_arm.py

```diff
@@ -0,0 +1,609 @@
+import logging
+import time
+import typing
+
+from bosdyn.api import arm_command_pb2
+from bosdyn.api import geometry_pb2
+from bosdyn.api import manipulation_api_pb2
+from bosdyn.api import robot_command_pb2
+from bosdyn.api import synchronized_command_pb2
+from bosdyn.api import trajectory_pb2
+from bosdyn.client.manipulation_api_client import ManipulationApiClient
+from bosdyn.client.robot import Robot
+from bosdyn.client.robot_command import (
+    RobotCommandBuilder,
+    RobotCommandClient,
+    block_until_arm_arrives,
+    blocking_stand,
+)
+from bosdyn.client.robot_state import RobotStateClient
+from bosdyn.client.time_sync import TimeSyncEndpoint
+from bosdyn.util import seconds_to_duration
+
+from wrapper import RobotState
```

Comment:
This is showing up as a circular dependency when I try to run it. wrapper.py imports spot_arm.py and spot_arm.py imports wrapper.py.